### PR TITLE
Add HTTPS capability

### PR DESCRIPTION
**Pull request by aarongerig**
As discussed today afternoon

### CODE REVIEW

File: PHP7.1-apache/Dockerfile [hunk 1]
```diff
@@ -54,10 +54,21 @@ RUN apt-get update && apt-get install -y lsb-release \
...
       && rm -rf advancecomp-1.17 \
     && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer \
+    \
+    && a2enmod rewrite \
+      && a2enmod ssl  \
+      && mkdir /etc/apache2/ssl \
+      && touch /etc/apache2/ssl/localhost.key \
+      && touch /etc/apache2/ssl/localhost.crt \
+      && openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/apache2/ssl/localhost.key \
+        -subj '/C=CH/ST=Lucerne/L=Sursee/O=w-vision AG/CN=localhost/' -out /etc/apache2/ssl/localhost.crt \
```
Comment (dpfaffenbauer) [inline PHP7.1-apache/Dockerfile]: No quite happy with that, can you change that?

**Review by dpfaffenbauer — CHANGES_REQUESTED** [review]

File: PHP7.1-apache/Dockerfile (hunk 1 above)
Comment (aarongerig) [reply PHP7.1-apache/Dockerfile]: What credentials shall I use?

Comment (dpfaffenbauer) [reply PHP7.1-apache/Dockerfile]: Mine :P 

No, just use something generic. Max Mustermans international brother or sister ;)

**Comment by dpfaffenbauer** [comment]
can you also add this to the other 2 images? php7-apache and php7.2-apache?

**Review by dpfaffenbauer — APPROVED** [review]

**Comment by dpfaffenbauer** [comment]
awesome 🎉

**Comment by aarongerig** [comment]
Phew! :)

**Pull request merged**